### ISSUE TITLE
Live site?

### ISSUE THREAD

**Issue by JacksonBates** [emmalearnscode/guide-to-HF#10]
Is this site available live anywhere, such as GitHub pages or Netlify?

It might help encourage new contributors if they could see their work go live on a real page, and gives a little demo of what to expect when it's up and running.

**Comment by sunnythepatel** [comment]
Hi, @JacksonBates and @emmalearnscode  I did host it using but I am not sure I am allowed to do that https://sunnythepatel.github.io/guide-to-HF/fork.html Let me know if its ok or not? If it's not I can remove it.

**Comment by emmalearnscode** [comment]
Please remove your link @sunnythepatel the site is already hosted on my github account

**Comment by emmalearnscode** [comment]
I’ll wait on the pr for the readme update then will raise this as another issue (just to avoid conflicts)
